doc/reference/devices_disk.md

```diff
@@ -70,6 +70,21 @@ VM `cloud-init`
 
       lxc config device add <instance_name> <device_name> disk source=cloud-init:config
 
+## Initial configuration for instance root disk devices
```

Comment:
Can we call this `Initial volume configuration for instance root disk devices` and then for all places that we say `Initial configuration` change to `Initial volume configuration`.